tensorflow/src/TensorFlow/Core.hs

```diff
@@ -0,0 +1,96 @@
+-- Copyright 2016 TensorFlow authors.
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+{-# LANGUAGE ExplicitNamespaces #-}
+
+-- | The core functionality of TensorFlow.
+--
+-- Unless you are defining ops, you do not need to import other modules from
+-- this package.
+--
+-- Ops are provided in the following modules:
+--
+--     * "TensorFlow.Ops"
+--     * "TensorFlow.GenOps.Core"
+--     * "TensorFlow.Gradient"
+--     * "TensorFlow.EmbeddingOps"
```

Comment:
Updated it to refer to the cabal packages instead of the modules. Added the docs in another branch: https://github.com/tensorflow/haskell/tree/simple_imports_docs

(I rebased onto HEAD without thinking, so the history of the PR is a little messed up)